include/experimental/__p0009_bits/macros.hpp

```diff
@@ -211,7 +211,7 @@
 //==============================================================================
 // <editor-fold desc="inline variables"> {{{1
 
-#if _MDSPAN_USE_INLINE_VARIABLES
+#ifdef _MDSPAN_USE_INLINE_VARIABLES
```

Comment:
If we really want to be safe on this we should do:

```suggestion
#if defined(_MDSPAN_USE_INLINE_VARIABLES) && _MDSPAN_USE_INLINE_VARIABLES != 0
```